app/config/security.yml

```diff
@@ -40,7 +40,8 @@ security:
             security: false
         admin_area:
             pattern:    ^/admin
-            http_basic: ~ 
+            http_basic: ~
+            logout_on_user_change: true
```

Comment:
Jeg er ikke helt oppdatert på hva som skjer i Symfony for tiden. Hvorfor trenger vi dette?

Reply:
![image](https://user-images.githubusercontent.com/46197518/97114427-bfc12380-16f0-11eb-89e5-4783cff5de0e.png)
Nytt fra Symfony 4 er at logout_on_user_change alltid er true.
I tillegg tar PR for seg denne deprecationen:
![image](https://user-images.githubusercontent.com/46197518/97114451-d798a780-16f0-11eb-8f8a-a9c6306ab68d.png)

Reply:
Okei, så etter å ha testet litt så betyr `logout_on_user_change` altså at brukere på **andre** devices blir automatisk logget ut hvis du endrer brukernavn eller passord. Sånn er det ikke i dag, men det er jo en fornuftig feature å slå på. Er det `User->isEqualTo` som bestemmer hvilke properties som blir sjekket for å bestemme om andre devices skal bli logget ut?